crates/proof-of-sql/src/proof_primitive/dory/public_parameters.rs

```diff
@@ -72,4 +72,21 @@ impl PublicParameters {
             blitzar_handle,
         }
     }
+
+    /// Get a reference to the blitzar handle.
+    #[cfg(feature = "blitzar")]
+    pub fn get_blitzar_handle(&self) -> &MsmHandle<ElementP2<ark_bls12_381::g1::Config>> {
```

Comment:
`get_` is not typical Rust
```suggestion
    pub(super) fn blitzar_handle(&self) -> &MsmHandle<ElementP2<ark_bls12_381::g1::Config>> {
```

Reply:
Removed, thank you.